source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -49,6 +53,23 @@ class EvaluationContext {
     set $data(value: any) {
         this._$data = value;
     }
+
+    generateVersionJson() {
+        const version = pkg.version;
+        const versionSplit = version.split('.');
+
+        let patchSplit = [];
```

Comment:
nit: null, nullptr or None might be better candidates.

Reply:
Why is it better to set to null?

Reply:
It's typescript and I think ` let patchSplit = []` is the correct way to avoid warnings, I believe you don't want to write code like
```
let patchSplit?: string[] = null
```
or 
```
let patchSplit: string[]  | undefined = undefined
```